hw3/slavrom/class.js

```diff
@@ -0,0 +1,82 @@
+import { SarahMay, JeremieBrown, Elizabeth } from '../people.data.js';
+
+class PersonClass {
+	constructor (person) {
+		this.person = person;
+	}
+
+	getFullName = () => {
+		const fullName = `${this.person.firstName} ${this.person.lastName}`;
+		try {
+			if (!this.person.firstName) {
```

Comment:
I think that if we have just firstName would be better to return this first name and log out warning to console that we don't have last name. But in case when we have just lastName it's completely fine to throw error